docs/jumpstart/week-10.md

```diff
@@ -0,0 +1,221 @@
+
+```shell
+git commit -m
+```
+which commits our added files
+
+We should already be logged into Heroku because we deployed our frontend code but if we're not, then we have to login again using:
+```shell
+heroku login
+```
+
+After logging in we run:
+
+```shell
+heroku create
+```
+which creates a new project on Heroku that represents our backend
+
+```shell
+git push heroku master
+```
+This will push our code to the Heroku server. Again, if we want to deploy a version that is not in our master branch we run the command:
+```shell
+git push heroku <branch_name>:master
+```
+
+After pushing our code to the Heroku server we should be given a URL. This URL is now the URL that our frontend needs to make request to.
+To verify that our backend deployed successfully, we again run the command:
+```shell
+heroku open
+```
+which should open a new browser window with a message at the top saying <strong> Hello Jumpstarters! </strong>
+
+Now that our backend and frontend are deployed, we have to configure our requests URL. We have to navigate back to our frontend project and modify our 
+<strong> .env.production </strong> file and change it from:
+
+~~REACT_APP_API_DOMAIN = "http://localhost:8081"~~
+REACT_APP_API_DOMAIN = "<heroku_backend_url>"
+
+Now that we changed our API request URL we have to re-deploy our frontend with this modified changes. To do this,
+we simply navigate back to our frontend project directory and type in:
+```shell
+git add .
+```
+Which adds all our modified files.
+
+```shell
+git commit -m "<message" .
+```
+
+Which commits our newly updated code.
+
+```shell
+git push heroku master
+```
+
+Will push the new changes to Heroku and update our current project.
+
+Now, our frontend application will make requests to our new request URL. 
+
+### Deploy PostgreSQL
+Okay so now we have deployed our frontend and our backend and now we have to deploy our database. Similar to our frontend and backend, our database has been running locally on our machine. Therefore, no one else has access to this data.
+We have to deploy our database to the cloud so that everyone that accesses our application shares the same database. 
+
+Heroku provides us many ways to configure a Postgres database for our application. in fact, Heroku can offer us a Postgres database that we can connect our application to. 
+
+First, we have to run our migration script. This is a command that will migrate our local database schema to our 
+Heroku postgresql database. To do this, we navigate to <strong> src/persist/resources/migrations </strong> directory.
+Once we are there, there should be a couple of SQL files. These SQL files are what will migrate our local database to our heroku
+postgresql database. To do this we run the command:
+```shell
+heroku pg:psql
+```
+This will start Heroku's postgres database that is provisioned to us. You should now see that our terminal says that we have connected to our postgresql database server. Now
```

Comment:
```suggestion
This will start a terminal connected to Heroku's Postgres database that is provisioned to us. You should now see that our terminal says that we have connected to our PostgreSQL database server. Now
```